src/test/java/org/opensearch/security/support/Base64HelperTest.java

```diff
@@ -43,54 +49,56 @@ private static Serializable ds(Serializable s) {
     public void testString() {
         String string = "string";
         Assert.assertEquals(string, ds(string));
+        Assert.assertEquals(string, dsJDK(string));
     }
 
     @Test
     public void testInteger() {
         Integer integer = Integer.valueOf(0);
         Assert.assertEquals(integer, ds(integer));
+        Assert.assertEquals(integer, dsJDK(integer));
     }
 
     @Test
     public void testDouble() {
         Double number = Double.valueOf(0.);
         Assert.assertEquals(number, ds(number));
+        Assert.assertEquals(number, dsJDK(number));
     }
 
     @Test
     public void testInetSocketAddress() {
         InetSocketAddress inetSocketAddress = new InetSocketAddress(0);
         Assert.assertEquals(inetSocketAddress, ds(inetSocketAddress));
-    }
-
-    @Test
```

Comment:
qq, why remove this test?

Reply:
We won't be supporting serialization of the `Pattern` class. I verified on my end and it seems that it's nowhere being serialised within security plugin. Similar is the case with `LdapEntry`.

The only classes that we directly serialise are -
* String
* UnmodifiableMap (for DLS queries, Masked Fields, FLS Fields)
* SourceFieldsContext
* User
* InjectedUser
* LdapUser
* InetSocketAddress

I've also raised an issue to check why ldap classes were present in the list of `Base64Helper::SAFE_CLASSES`. 
https://github.com/opensearch-project/security/issues/2800